src/Framework/ConversationTopic.cs

```diff
@@ -0,0 +1,54 @@
+﻿using System;
+using StardewModdingAPI;
+using StardewValley;
+
+
+namespace QuestFramework.Framework
+{
+    public class ConversationTopicOptions
+    {
+        internal string AddWhenQuestAccepted { get; set; }
+        internal string AddWhenQuestRemoved { get; set; }
+        internal string AddWhenQuestCompleted { get; set; }
+        internal string RemoveWhenQuestCompleted { get; set; }
+        internal string RemoveWhenQuestRemoved { get; set; }
+        internal string RemoveWhenQuestAccepted { get; set; }
+    }
+
+    internal class ConversationTopicMethods
```

Comment:
Join this class with the `ConversationTopicOptions` and leave name of that class as `ConversationTopic` in namespace `QuestFramework.Framework.ContentPacks`